### PR TITLE
Update master-node-communication.md

### PR DESCRIPTION
**Pull request by dgabrysch**
Fixing Typo, adding small linguistic changes.

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by dgabrysch** [comment]
Signed up.

File: content/de/docs/concepts/architecture/master-node-communication.md [hunk 1]
```diff
@@ -63,10 +63,9 @@ Diese Verbindungen **sind derzeit nicht sicher** innerhalb von nicht vertrauensw
 
 ### SSH Tunnels
 
-Kubernetes unterstützt SSH-Tunnel zum Schutz der Kommunikationspfade von Master -> Cluster.
-In dieser Konfiguration initiiert der Apiserver einen SSH-Tunnel zu jedem Nodem im Cluster (Verbindung mit dem SSH-Server, der Port 22 läuft), und leitet den gesamten Datenverkehr für ein kubelet, einen Node, einen Pod oder einen Dienst durch den Tunnel.
-Dieser Tunnel stellt sicher, dass der Datenverkehr nicht außerhalb des Netzwerks sichtbar ist, in dem die Knoten ausgeführt werden.
-
-SSH-Tunnel sind derzeit nicht mehr unterstützt. Sie sollten sie also nicht verwenden, es sei denn, Sie wissen, was Sie tun. Ein Ersatz für diesen Kommunikationskanal wird entwickelt.
+Kubernetes unterstützt SSH-Tunnel zum Schutz der Kommunikationspfade von Mastern in Richtung des Clusters.
```
Comment (mkorbi) [inline content/de/docs/concepts/architecture/master-node-communication.md]: By checking the origin: "Kubernetes supports SSH tunnels to protect the Master -> Cluster communication paths." I would change it to:  "Kubernetes unterstützt SSH-Tunnel zum Schutz der Master -> Cluster Kommunikationspfade."

File: content/de/docs/concepts/architecture/master-node-communication.md [hunk 2]
```diff
@@ -63,10 +63,9 @@ Diese Verbindungen **sind derzeit nicht sicher** innerhalb von nicht vertrauensw
...
+In dieser Konfiguration erstellt der Apiserver einen SSH-Tunnel zu jedem Node im Cluster (Verbindung mit dem SSH-Server, der Port 22 läuft), und leitet den gesamten Datenverkehr für ein kubelet, einen Node, einen Pod oder einen Dienst durch den Tunnel.
+Dieser Tunnel stellt sicher, dass der Datenverkehr nicht außerhalb des Netzwerks sichtbar ist, in dem die Knoten laufen.
+SSH-Tunnel zur Zeit nicht unterstützt und sollten nicht genutzt werden, es sei denn, Sie wissen, was Sie tun. Ein Ersatz für diesen Kommunikationskanal wird entwickelt.
```
Comment (mkorbi) [inline content/de/docs/concepts/architecture/master-node-communication.md]: Please add: SSH-Tunnel sind zur Zeit nicht ...

File: content/de/docs/concepts/architecture/master-node-communication.md [hunk 3]
```diff
@@ -63,10 +63,9 @@ Diese Verbindungen **sind derzeit nicht sicher** innerhalb von nicht vertrauensw
 
 ### SSH Tunnels
 
-Kubernetes unterstützt SSH-Tunnel zum Schutz der Kommunikationspfade von Master -> Cluster.
-In dieser Konfiguration initiiert der Apiserver einen SSH-Tunnel zu jedem Nodem im Cluster (Verbindung mit dem SSH-Server, der Port 22 läuft), und leitet den gesamten Datenverkehr für ein kubelet, einen Node, einen Pod oder einen Dienst durch den Tunnel.
-Dieser Tunnel stellt sicher, dass der Datenverkehr nicht außerhalb des Netzwerks sichtbar ist, in dem die Knoten ausgeführt werden.
-
-SSH-Tunnel sind derzeit nicht mehr unterstützt. Sie sollten sie also nicht verwenden, es sei denn, Sie wissen, was Sie tun. Ein Ersatz für diesen Kommunikationskanal wird entwickelt.
+Kubernetes unterstützt SSH-Tunnel zum Schutz der Kommunikationspfade von Mastern in Richtung des Clusters.
+In dieser Konfiguration erstellt der Apiserver einen SSH-Tunnel zu jedem Node im Cluster (Verbindung mit dem SSH-Server, der Port 22 läuft), und leitet den gesamten Datenverkehr für ein kubelet, einen Node, einen Pod oder einen Dienst durch den Tunnel.
```
Comment (mkorbi) [inline content/de/docs/concepts/architecture/master-node-communication.md]: I would stuck with "initiiert" as the API server triggers the creation of the ssh tunnel but the API server does not create the ssh tunnel in terms of setup the tunnel.

Comment (dgabrysch) [reply content/de/docs/concepts/architecture/master-node-communication.md]: Seems right to me:)

File: content/de/docs/concepts/architecture/master-node-communication.md (hunk 1 above)
Comment (dgabrysch) [reply content/de/docs/concepts/architecture/master-node-communication.md]: You are right, it is more like the origin.

File: content/de/docs/concepts/architecture/master-node-communication.md (hunk 2 above)
Comment (dgabrysch) [reply content/de/docs/concepts/architecture/master-node-communication.md]: Ok, I habe overseen that will change it

Comment (dgabrysch) [reply content/de/docs/concepts/architecture/master-node-communication.md]: Though, normally Inwould something like „werden zur Zeit nicht unterstützt“ - but that would not quite be the translation of the original.

File: content/de/docs/concepts/architecture/master-node-communication.md [hunk 4]
```diff
@@ -41,19 +41,19 @@ Der zweite ist vom Apiserver zu einem beliebigen Node, Pod oder Dienst über die
...
+Um diese Verbindung zu überprüfen, verwenden Sie das Flag `--kubelet-certificate-authority`, um dem Apiserver ein Stammzertifikatbündel bereitzustellen, das zur Überprüfung des Server-Zertifikats des kubelets verwendet wird.
 
 Wenn dies nicht möglich ist, verwenden Sie [SSH tunneling](/docs/concepts/architecture/master-node-communication/#ssh-tunnels)
-zwischen dem Apiserver und kubelet, falls erforderlich, um eine Verbindung über ein nicht vertrauenswürdiges oder öffentliches Netz zu vermeiden.
+zwischen dem Apiserver und dem kubelet, falls es erforderlich ist eine Verbindung über ein nicht vertrauenswürdiges oder öffentliches Netz zu vermeiden.
 
-Ausserdem sollte, [Kubelet Authentifizierung und/oder Autorisierung](/docs/admin/kubelet-authentication-authorization/) sollte aktiviert sein, um die kubelet-API zu sichern.
+Außerdem sollte [Kubelet Authentifizierung und/oder Autorisierung](/docs/admin/kubelet-authentication-authorization/) aktiviert sein, um die kubelet-API zu absichern.
```
Comment (mkorbi) [inline content/de/docs/concepts/architecture/master-node-communication.md]: zu sichern oder abzusichern aber nicht zu absichern :)

Comment (dgabrysch) [reply content/de/docs/concepts/architecture/master-node-communication.md]: Right :)

Comment (dgabrysch) [reply content/de/docs/concepts/architecture/master-node-communication.md]: I am still thinking what I tried to do there, guess it was autocorrect (which I should turn off now :D )

**Comment by mkorbi** [comment]
Thank you, and Merry Christmas! :) 

/lgtm
/approve

**Pull request merged**